modules/redis/src/main/scala/zio/redis/Output.scala

```diff
@@ -727,10 +727,26 @@ object Output {
     }
   }
 
+  case object ClientInfoOutput extends Output[ClientInfo] {
+    protected def tryDecode(respValue: RespValue): ClientInfo =
+      respValue match {
+        case RespValue.BulkString(s) => ClientInfo.from(s.asString)
+        case other                   => throw ProtocolError(s"$other isn't a bulk string")
+      }
+  }
+
+  case object ClientsInfoOutput extends Output[ClientsInfo] {
+    protected def tryDecode(respValue: RespValue): ClientsInfo =
+      respValue match {
+        case RespValue.BulkString(s) => ClientInfo.from(s.asString.split("\r\n").filter(_.nonEmpty))
+        case other                   => throw ProtocolError(s"$other isn't a bulk string")
+      }
+  }
```

Comment:
Let's rename this to `ClientListOutput`. Also prefer `Chunk` to `List`, e.g.:

```scala
case object ClientListOutput extends Output[Chunk[ClientInfo]] { ... }
```

You might be able to reuse `ClientInfoOutput` in its implementation, though it's not necessary.